<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Status</class>
 <widget class="QWidget" name="Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="ratePlotGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Rate</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="2">
         <widget class="QPushButton" name="resetRatePushButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="PlotCustom" name="ratePlot" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="ratePlotPresetComboBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ratePlotBufferEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pulseHeightHistogramGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>130</height>
        </size>
       </property>
       <property name="title">
        <string>Pulse Height Histogram</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="4">
         <widget class="CustomHistogram" name="pulseHeightHistogram" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="histoLogYCheckBox">
          <property name="text">
           <string>log Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="PulseHeightHistogramEntriesLabel">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0 entries</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="resetHistoPushButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="statusValuesGroupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>190</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>190</height>
      </size>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="ADCGroupBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>ADC Voltages</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="ADCLabel1">
               <property name="text">
                <string>Ch1: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ADCLabel2">
               <property name="text">
                <string>Ch2: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ADCLabel3">
               <property name="text">
                <string>Ch3: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ADCLabel4">
               <property name="text">
                <string>Ch4: </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="DACGroupBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>110</height>
           </size>
          </property>
          <property name="title">
           <string>DAC Voltages</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="DACLabel1">
               <property name="text">
                <string>Ch1: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="DACLabel2">
               <property name="text">
                <string>Ch2: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="DACLabel3">
               <property name="text">
                <string>Ch3: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="DACLabel4">
               <property name="text">
                <string>Ch4: </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="switchesGroupBox">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="title">
         <string>Switches</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="InputSelectGroupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>144</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>110</height>
            </size>
           </property>
           <property name="title">
            <string>Timing Input</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <widget class="QRadioButton" name="InputSelectRadioButton0">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>25</y>
              <width>69</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>AND</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton1">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>45</y>
              <width>69</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>XOR</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton2">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>65</y>
              <width>69</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>DISCR1</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton3">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>85</y>
              <width>69</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>DISCR2</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton4">
            <property name="geometry">
             <rect>
              <x>87</x>
              <y>25</y>
              <width>45</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IN4</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton5">
            <property name="geometry">
             <rect>
              <x>87</x>
              <y>45</y>
              <width>45</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IN5</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton6">
            <property name="geometry">
             <rect>
              <x>87</x>
              <y>65</y>
              <width>45</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IN6</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="InputSelectRadioButton7">
            <property name="geometry">
             <rect>
              <x>87</x>
              <y>85</y>
              <width>45</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IN7</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="biasEnableCheckBox">
             <property name="text">
              <string>Bias On</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="highGainCheckBox">
             <property name="text">
              <string>High Gain</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preamp1CheckBox">
             <property name="text">
              <string>Preamp 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="preamp2CheckBox">
             <property name="text">
              <string>Preamp 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>52</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="temperatureLabel">
        <property name="text">
         <string>Temperature:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotCustom</class>
   <extends>QWidget</extends>
   <header>plotcustom.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomHistogram</class>
   <extends>QWidget</extends>
   <header>custom_histogram_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
